source/simulator/assembly.cc

```diff
@@ -828,6 +831,8 @@ namespace aspect
             scratch.old_pressure);
         scratch.finite_element_values[introspection.extractors.pressure].get_function_values (old_old_solution,
             scratch.old_old_pressure);
+        scratch.finite_element_values[introspection.extractors.pressure].get_function_gradients (current_linearization_point,
+            scratch.current_pressure_gradients);
```

Comment:
I believe we did this because we thought it was needed (but the differences from this being in or not in was not enough to cause the tests to fail, if I recall).   Would you like it removed?